firebase-analytics/documentation.md

```diff
@@ -0,0 +1,2 @@
+# Module firebase-analytics
```

Comment:
can we use the official documentation module introductions here? they should be found in the package-info.java files in the android sdk

Reply:
Yes sure! I will change that :)

Reply:
The package-info.java files in the android sdk's don't seem to contain the module introductions that you mention. Where did you find that?

Reply:
should we take the ones from the android sdk?
https://github.com/firebase/firebase-android-sdk/tree/main/docs/ktx

Reply:
Thats only documentation for the kotlin extensions so i think thats not quite right

Reply:
I assumed they would have intros in the packages, but they don't they are just empty - you can see here https://firebase.google.com/docs/reference/android/com/google/firebase/firestore/package-summary

We can just leave ours empty too in this case

Reply:
okay! ill leave it like this then